src/main/java/io/quarkus/gizmo2/impl/ClassCreatorImpl.java

```diff
@@ -19,6 +19,7 @@
 public final class ClassCreatorImpl extends TypeCreatorImpl implements ClassCreator {
     public ClassCreatorImpl(final ClassDesc type, final ClassBuilder zb) {
         super(type, zb, 0);
+        withFlags(AccessFlag.PUBLIC, AccessFlag.SYNTHETIC);
```

Comment:
I guess we'll also need a `withoutFlags` then to override the default. WDYT?

Reply:
Oh, here's an idea, what about methods like `cc.public_();` `cc.packagePrivate()` etc.? They could intelligently set/clear flags as needed.